### ISSUE TITLE
Prevent Nerd Poling

### ISSUE THREAD

**Issue by ikabod-kee** [PixelGuys/Cubyz#851]
When the player is airborne, they gain a hitbox under them that prevents blocks from being placed directly below them.
Additionally (and optionally,) if methods are found to circumvent this, you can have the player unable to place blocks if they're looking down at a certain angle.

Should also only be enabled in Survival mode

![image](https://github.com/user-attachments/assets/daef2686-1566-41ed-9cb9-2b77067d7b97)

**Comment by careeoki** [comment]
Most of the annoyances from this would be solved with #727
In the future we could have an accessory that allows the creative mode functionality, at the cost of 50% less damage with weapons, or something.

**Comment by thecreare** [comment]
I don't know if this is still something thats planned, but if it is, here is why I don't think it works and what I think should be done instead:

- It seems very trivial to bypass, building a 2x1 tower and walking back and forth between the blocks (no jumping+building required)
- I think the remaining annoyances after #727 would still outweigh the minimal benefits (stopping one of many ways to cheese mobs)
- I saw scaffolding mentioned in Discord as a way to not effect building with this change. Scaffolding (assuming minecraft-like) could be used to cheese by building a tower, climbing to the top, placeing a block to the side, standing on that, then destroying the scaffolding. Preventing enemies from reaching you.

There are so many ways to cheese minecraft-like mobs with blocks. I think the fundamental concept of a sandbox game where you can modify a world that the enemies can't gives you an unfixable advantage.

Since I've heard that defeating enemies in an area would clear it forever, why not allow enemies to build and destroy blocks? There was an awesome Minecraft mod that did this and its incredibly fun. The problem is it takes over the whole game because most of your time is spent constantly repairing your base from respawning mobs. If enemies don't respawn, this would fix every easy cheese without that problem.

*I can't tell if this came across rude, if It did I'm sorry I didn't meant it that way*

**Comment by ikabod-kee** [comment]
> why not allow enemies to build and destroy blocks?

I absolutely agree with this! My main concern was skybases actually XD

**Comment by ikabod-kee** [comment]
But, I suppose nerd poling should stay. We'll just design around the fact that players can build whatever they want.

**Comment by Minhoquinha** [comment]
I disagree with this, I don't think just making a large pole to stay on top and avoid danger should be nerfed. I think using flying enemies would be a more senseful way to counter easy cheese like this. The breaking blocks enemies would be good against nerd digging (carving out a hole on the ground and staying there to avoid enemies).

Or better yet... What about a sniper enemy? A very slow and fragile enemy that relies completely on direct sight, it doesn't spawn frequently but lurks around at night, when it sees the player it shoots them down with a heavy projectile with powerful knockback. As a sniper it would be capable of detecting the player from veeery far (we are talking over 300 blocks). It would generally not be a large threat, fragile, slow and uncommon, but when the player nerd polls, they open up many different lines of sight from far, almost guaranteeing that a far away sniper can see you.
